### PR TITLE
updated code for build,release & agent pool service id mapping

### PR DESCRIPTION
**Pull request by anandsinghms**
## Description
</br>
 Brief description of the requested change.

## Checklist
- [ ] I have read the instructions mentioned in [Contribute to Code](/Contributing.md).
- [ ] I have read and understood the criteria described under [submitting changes](/Contributing.md#submitting-changes). 
- [ ] The title of the PR clearly describes the intent of the PR.
- [ ] This PR does not introduce any breaking changes to the code.

### CODE REVIEW

File: src/AzSK.ADO/Framework/Core/STMapping/AzSKADOServiceMapping.ps1 [hunk 1]
```diff
@@ -74,20 +78,92 @@ class AzSKADOServiceMapping: CommandBase
...
                 $this.ProjectId = $this.ReleaseSTDetails.data[0].projectId
             }
+        }       
+
+        # Get Release-Repo mappings
+        try {                         
+            $releaseObjectListURL = ("https://vsrm.dev.azure.com/{0}/{1}/_apis/release/definitions?api-version=6.0" ) -f $($this.orgName), $this.projectName;    
+            $releaseObjectList = $this.GetBuildReleaseObjects($ReleaseObjectListURL,'Release');
+            $counter =0
+            foreach ($release in $releaseObjectList) {
```
Comment (arvindsingh-cloud) [inline src/AzSK.ADO/Framework/Core/STMapping/AzSKADOServiceMapping.ps1:127]: run loop for unmapped releases only same as build

File: src/AzSK.ADO/Framework/Core/STMapping/AzSKADOServiceMapping.ps1 [hunk 2]
```diff
@@ -37,6 +37,10 @@ class AzSKADOServiceMapping: CommandBase
             if((Test-Path $this.BuildMappingsFilePath) -and (Test-Path $this.ReleaseMappingsFilePath))
             {
                 $this.GetBuildReleaseMapping();
+                if ([string]::IsNullOrWhiteSpace($this.MappingType) -or $this.MappingType -eq "All" -or $this.MappingType -eq "Build")
+                {
+                    $this.FetchPipelineConnMapping();
```
Comment (arvindsingh-cloud) [inline src/AzSK.ADO/Framework/Core/STMapping/AzSKADOServiceMapping.ps1:42]: there are some issue in these two methods FetchPipelineConnMapping and FetchSvcConnMapping why both the methods are generating serviceconnection ST files

File: src/AzSK.ADO/Framework/Core/STMapping/AzSKADOServiceMapping.ps1 (hunk 1 above)
Comment (anandsinghms) [reply src/AzSK.ADO/Framework/Core/STMapping/AzSKADOServiceMapping.ps1:121]: done

File: src/AzSK.ADO/Framework/Core/STMapping/AzSKADOServiceMapping.ps1 (hunk 2 above)
Comment (anandsinghms) [reply src/AzSK.ADO/Framework/Core/STMapping/AzSKADOServiceMapping.ps1]: added for testing, removed extra one

**Review by arvindsingh-cloud — APPROVED** [review]

**Review by arvindsingh-cloud — APPROVED** [review]

**Pull request merged**